client/src/app/interfaces/task.ts

```diff
@@ -0,0 +1,8 @@
+export interface ITask {
+  id: number;
+  title: string;
+  menteeId: number;
+  description: string;
+  status: string;
+  dueDate: Date;
```

Comment:
Already commented on the server-side commit, but I don't think you can send a JavaScript `Date` object to the server? Am I wrong?